golos.publication/golos.publication.abi

```diff
@@ -614,6 +624,96 @@
             "type": "limitsarg"
         }
     ]
+  },
+  {
+    "name": "st_max_vote_changes",
+    "base": "",
+    "fields": [
+      {
+        "name": "max_vote_changes",
```

Comment:
this variant parameter in action looks like: `["st_max_vote_changes":{"max_vote_changes":123}]` when sending in action. there is no reason to have `"max_vote_changes"` as field name, because it's already in variant name. 
Propose change field name to simple `"value"`, so parameter become `["st_max_vote_changes":{"value":5}]`. Same with other single-field parameters.